posthog/api/__init__.py

```diff
@@ -15,11 +16,24 @@
 )
 
 router = routers.DefaultRouter()
+
+# TODO: add other checks necessary for using EE code
```

Comment:
Yeah this somehow needs to check whether there is a currently active license

Reply:
Yeah, we need to add the necessary checks here before merging

Reply:
is this a blocker for loading events into CH?

Reply:
Going to ping here just for convenience and will go check myself: @timgl has license checking been applied on any EE feature yet?

Reply:
Normally you can do it via `user.feature_available()` which we've used once for zapier. However in this case it needs to check either
1. whether it's multi tenancy or
2. whether there are _any_ active License objects on this instance

Reply:
BTW I don't think this is a blocker for getting this out